rules.txt

```diff
@@ -228,6 +228,8 @@
 /posts/office-of-sustainability/2017-03-28-environment-sustainability-city-of-philadelphia-action-guide 301 /2017-11-29-environmental-protection-during-the-trump-administration/
 /potholes                                 301  http://www.philadelphiastreets.com
 /ppr                                      301  /departments/philadelphia-parks-recreation/
+/ParksandRecreation/Pages/(.*)            301  /departments/philadelphia-parks-recreation/  #Handle department homepage

```

Comment:
Hm, I think this case should be covered by the next one, shouldn't it?

Reply:
Lol, yeah when you're being logical about it.